### DISCUSSION TITLE
Address family not supported by protocol

### DISCUSSION THREAD

**Discussion by rafawalter** [cypress-io/cypress#30277] [Docker]
Any ideas on fixing the error below?

`ERROR:socket_posix.cc(93)] CreatePlatformSocket() failed: Address family not supported by protocol (97)`

Error happens when running in Docker/IC Ubuntu 22.04. Works fine in Windows localhost

Already tried `export NODE_OPTIONS=--dns-result-order=ipv4first` without success

Cypress:        13.14.2
Browser:        Electron 118 (headless)
Node Version:   v20.17.0 (/usr/share/node-v20.17.0-linux-x64/bin/node)

**Comment by jpierson-at-riis** [comment]
Sounds a possible duplicate of https://github.com/cypress-io/cypress/issues/30123
